docs/protocol.md

```diff
@@ -72,23 +72,23 @@ See [DID Create API](#DID-Creation) section for detail on how to construct a cre
+e.g. `did:sidetree:<unique-portion>?-<method-name>-initial-state=<encoded-create-operation-request>`.
 
 This allows any entity to support all of the following usage patterns:
 
 - Resolving unpublished DIDs.
 - Authenticating with unpublished DIDs.
 - Signing and verifying credentials signed against unpublished DIDs.
-- Authenticating with either the DID or DID with `initial-values` parameter, after it is published.
-- Signing and verifying credentials signed against either the DID or DID with `initial-values` parameter, after it is published.
+- Authenticating with either the DID or DID with `-<method-name>-initial-state` parameter, after it is published.
+- Signing and verifying credentials signed against either the DID or DID with `-<method-name>-initial-state` parameter, after it is published.
 
 ### `published` Flag
 
-At such time an ID is published/anchored, a user can provide either the parametered or unparametered version of the ION DID URI to an external party, and it will be resolvable. There is no required change for any party that had been holding the parametered version of the URI - it will continue to resolve just as it had prior to being anchored. In addition, the community will introduce a generic, standard property: `published` in the [DID resolution spec](https://w3c-ccg.github.io/did-resolution/#output-resolvermetadata), that is added to the DID resolution response. The `published` property indicates whether a DID has been published/anchored in the underlying trust system a DID Method writes to. When an entity resolves any DID from any DID Method and finds that the DID has been published, the entity may drop the `initial-values` DID parameter from their held references to the DID in question, if they so desire. However, dropping the `initial-values` DID parameter after publication is purely an elective act - the ID will resolve correctly regardless.
+At such time an ID is published/anchored, a user can provide either the parametered or unparametered version of the ION DID URI to an external party, and it will be resolvable. There is no required change for any party that had been holding the parametered version of the URI - it will continue to resolve just as it had prior to being anchored. In addition, the community will introduce a generic, standard property: `published` in the [DID resolution spec](https://w3c-ccg.github.io/did-resolution/#output-resolvermetadata), that is added to the DID resolution response. The `published` property indicates whether a DID has been published/anchored in the underlying trust system a DID Method writes to. When an entity resolves any DID from any DID Method and finds that the DID has been published, the entity may drop the `-<method-name>-initial-state` DID parameter from their held references to the DID in question, if they so desire. However, dropping the `-<method-name>-initial-state` DID parameter after publication is purely an elective act - the ID will resolve correctly regardless.
```

Comment:
Noted in https://github.com/decentralized-identity/sidetree/pull/468 that `ION` should be generic (presume `Sidetree`).